internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -883,12 +883,26 @@ components:
       - $ref: '#/components/schemas/GCPUploadOptions'
       - $ref: '#/components/schemas/AzureUploadOptions'
       - $ref: '#/components/schemas/ContainerUploadOptions'
+      - $ref: '#/components/schemas/LocalUploadOptions'
       description: |
         This should really be oneOf but AWSS3UploadOptions is a subset of
         AWSEC2UploadOptions. This means that all AWSEC2UploadOptions objects
         are also valid AWSS3UploadOptionas objects which violates the oneOf
         rules. Therefore, we have to use anyOf here but be aware that it isn't
         possible to mix and match more schemas together.
+    LocalUploadOptions:
+      type: object
+      additionalProperties: false
+      required:
+        - local-save
```

Comment:
Can you make this `local_save`? We are pretty consistent with having `_` as the delimiter in API keys.

Reply:
Sure!